python/dgl/graphbolt/impl/gpu_cached_feature.py

```diff
@@ -8,6 +8,11 @@
 __all__ = ["GPUCachedFeature"]
 
 
+def nbytes(tensor):
```

Comment:
@Rhett-Ying nbytes reimplemented here for torch < 2.1.

Reply:
@mfbalin please add a comment here to clarify why we define it on our own. So when we do clean-up and totally turn to > 2.0 in near future, we could remove it directly.